wbooks/src/app/screens/Library/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { FlatList, View } from 'react-native';
+
+import { BOOKS_MOCK } from '@constants/mockBooks';
+
+import Book from './components/Book';
+import { AppColors } from '@constants/colors';
+
+function Library() {
+  return (
```

Comment:
create constants functions before this line to avoid creating them inside the return. This goes for the renderItem and the keyExtractor